data/en/lms_2.json

```diff
@@ -909,7 +912,10 @@
                                 "unit": "duration-year",
```

Comment:
I know it's not part of this, but didn't want to create a separate PR, line 894 above (description in titles) should just be value, I think it can be removed as it doesn't get displayed?